src/traefik.py

```diff
@@ -199,14 +199,20 @@ def _update_static_configuration(self):
         tcp_entrypoints = self._tcp_entrypoints
         logger.debug(f"Statically configuring traefik with tcp entrypoints: {tcp_entrypoints}.")
 
+        web_config = {
+            "address": f":{self.port}",
+            # enable by default http -> https redirect
+            "http": {"redirections": {"entryPoint": {"to": "websecure", "scheme": "https"}}},
```

Comment:
Might be missing something, but should we add a redirection scheme only if there is a certificate? I could imagine a tricky redirect look for a HTTP-only setup if traefik doesn't take into account whether certificate exists.

Reply:
that might explain why the itests are failing though :D